internal/pkg/middleware/middleware.go

```diff
@@ -574,6 +574,22 @@ func (m *Middleware) GetDefaultProject(r *http.Request, projectRepo datastore.Pr
 	return projects[0], err
 }
 
+func (m *Middleware) RequirePersonalAccessToken() func(next http.Handler) http.Handler {
```

Comment:
@danvixent JWTs are authenticated through the native realm. Will it go through this?

Reply:
@Subomi it won't, we check the AuthenticatedBy field.

Reply:
@Subomi they are actually authenticated by the JWT realm

Reply:
Make sense. @danvixent 

How can we ensure it's only PATs that can access this route, not all API Keys authenticated by the native realm.